example_running_scripts/locate.py

```diff
@@ -33,10 +33,10 @@
 response_inv = read_response_inv(response_file)
 
 # --- Create new Archive and set path structure ---
-archive = Archive(stations=stations, archive_path=archive_path)
-archive.path_structure(archive_format="YEAR/JD/STATION")
+archive = Archive(archive_path=archive_path, stations=stations,
+                  archive_format="YEAR/JD/STATION")
 # For custom structures...
-# archive.format = "custom/archive_{year}_{month}_structure"
+# archive.format = "custom/archive_{year}_{jday}/{month:02d}-{day:02d}.{station}_structure"
```

Comment:
On second thoughts, might it be sensible to remove this explicit casting, just in case there are weird data formats where julday/month are not 0 padded? Might just make things more of a faff for the majority of formats...

Reply:
I agree: I think the hardcoding in _load_from_path() is not necessary. The `YearMonthDay` option in `path_structure()` already still has the `:02d` in, which is maybe where my confusion originated.

Theoretically could it be extended to str as well as integer representations of the times if `now.month` was passed instead of a pre-formatted string? It would require using `strftime()` instead of simple string formatting, I guess.

Reply:
Resolved in https://github.com/QuakeMigrate/QuakeMigrate/pull/71/commits/0bb07c56eda9440f6a1c71a21bce48558adcfd00